CDCgov/prime-reportstream
# 8552 Gracefully Handle Routing Filters when a property isn't present in the FHIR Bundle

This PR changes the default filters to avoid exceptions when a FHIR bundle property being queried doesn't exist, and reduces the log level of query exceptions so that on call is paged less frequently.

Test Steps:
1. Remove the value of `performerState` or `patientState` from a UP submission. The shorthand definitions can be viewed in `fhirpath_filter_shorthand.csv`.
2. Observe the UP routing function and confirm either an exception is not thrown (if the `organization.yml` updates are in place) or that an exception is written as a warning instead of an error.

## Changes
- Changed the handling of a `SchemaException` in the UP routing function to write a `warn` in the actionLogger instead of `error`.
- Changed the handling of the same so that the `SchemaException` is not rethrown after being logged. Also changed the handling of a call to the query function to not expect an exception to be thrown in this case.
- Renamed the actionLogger message object to better describe the scenario.
- Updated the default queries defined for full ELR receivers to handle the case that the target object may not exist without throwing an exception.

## Checklist

### Testing

## Linked Issues
- Fixes #8552
